templates/spec/rails_helper.rb

```diff
@@ -1,7 +1,16 @@
-require 'simplecov'
-
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
+
+# Coverage
+require 'simplecov'
+# save to CircleCI's artifacts directory if we're on CircleCI
+if ENV['CIRCLE_ARTIFACTS']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.